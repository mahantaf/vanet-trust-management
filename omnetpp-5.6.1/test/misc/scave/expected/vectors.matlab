offered_load.description="offered load; Net.subnet[0]; testfiles/vectors.vec; "
offered_load.X=[
0.1;
0.2;
0.3;
0.4;
0.5;
0.6;
0.7;
0.8;
0.9;
1;
1.1;
1.2;
1.3;
1.4;
1.5;
1.6;
1.7;
1.8;
1.9;
2;
]

offered_load.Y=[
30;
40;
10;
20;
50;
60;
70;
80;
90;
100;
110;
120;
130;
140;
150;
160;
170;
180;
190;
200;
]

throughput____.description="throughput (%); Net.subnet[0]; testfiles/vectors.vec; "
throughput____.X=[
0.1;
0.2;
0.3;
0.41;
0.51;
0.6;
0.7;
0.8;
0.9;
1;
1.1;
1.2;
1.3;
1.4;
1.5;
1.6;
1.7;
1.8;
1.9;
2;
]

throughput____.Y=[
35.11;
45.06;
8.12;
22.96;
53.21;
59.88;
65.34;
69.81;
73.47;
76.46;
78.91;
80.92;
82.57;
83.91;
84.5;
83.92;
80.86;
74.87;
68.77;
57.16;
]

frames_dropped____.description="frames dropped (%); Net.subnet[0]; testfiles/vectors.vec; "
frames_dropped____.X=[
0.1;
0.21;
0.31;
0.41;
0.51;
0.6;
0.7;
0.8;
0.9;
1;
1.1;
1.2;
1.3;
1.4;
1.5;
1.6;
1.7;
1.8;
1.9;
2;
]

frames_dropped____.Y=[
0;
0;
0;
0;
0;
0.2;
0;
0;
1.2;
0;
0.9;
1.4;
2.1;
2.2;
2.6;
5.1;
9.2;
14.4;
25.4;
43.4;
]

