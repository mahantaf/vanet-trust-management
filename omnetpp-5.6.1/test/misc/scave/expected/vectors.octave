# name: offered_load
# type: struct
# length: 3
# name: description
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 1
# length: 52
offered load; Net.subnet[0]; testfiles/vectors.vec;
# name: X
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0.1
0.2
0.3
0.4
0.5
0.6
0.7
0.8
0.9
1
1.1
1.2
1.3
1.4
1.5
1.6
1.7
1.8
1.9
2
# name: Y
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
30
40
10
20
50
60
70
80
90
100
110
120
130
140
150
160
170
180
190
200
# name: throughput____
# type: struct
# length: 3
# name: description
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 1
# length: 54
throughput (%); Net.subnet[0]; testfiles/vectors.vec;
# name: X
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0.1
0.2
0.3
0.41
0.51
0.6
0.7
0.8
0.9
1
1.1
1.2
1.3
1.4
1.5
1.6
1.7
1.8
1.9
2
# name: Y
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
35.11
45.06
8.12
22.96
53.21
59.88
65.34
69.81
73.47
76.46
78.91
80.92
82.57
83.91
84.5
83.92
80.86
74.87
68.77
57.16
# name: frames_dropped____
# type: struct
# length: 3
# name: description
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 1
# length: 58
frames dropped (%); Net.subnet[0]; testfiles/vectors.vec;
# name: X
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0.1
0.21
0.31
0.41
0.51
0.6
0.7
0.8
0.9
1
1.1
1.2
1.3
1.4
1.5
1.6
1.7
1.8
1.9
2
# name: Y
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0
0
0
0
0
0.2
0
0
1.2
0
0.9
1.4
2.1
2.2
2.6
5.1
9.2
14.4
25.4
43.4
