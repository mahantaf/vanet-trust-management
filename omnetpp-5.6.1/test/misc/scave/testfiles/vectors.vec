# generated by scave

vector 0  Net.subnet[0]  "offered load"  TV
vector 1  Net.subnet[0]  "throughput (%)"  TV
vector 2  Net.subnet[0]  "frames dropped (%)"  TV
0	0.1	30
0	0.2	40
0	0.3	10
0	0.4	20
0	0.5	50
0	0.6	60
0	0.7	70
0	0.8	80
0	0.9	90
0	1	100
0	1.1	110
0	1.2	120
0	1.3	130
0	1.4	140
0	1.5	150
0	1.6	160
0	1.7	170
0	1.8	180
0	1.9	190
0	2	200
1	0.1	35.11
1	0.2	45.06
1	0.3	8.12
1	0.41	22.96
1	0.51	53.21
1	0.6	59.88
1	0.7	65.34
1	0.8	69.81
1	0.9	73.47
1	1	76.46
1	1.1	78.91
1	1.2	80.92
1	1.3	82.57
1	1.4	83.91
1	1.5	84.5
1	1.6	83.92
1	1.7	80.86
1	1.8	74.87
1	1.9	68.77
1	2	57.16
2	0.1	0
2	0.21	0
2	0.31	0
2	0.41	0
2	0.51	0
2	0.6	0.2
2	0.7	0
2	0.8	0
2	0.9	1.2
2	1	0
2	1.1	0.9
2	1.2	1.4
2	1.3	2.1
2	1.4	2.2
2	1.5	2.6
2	1.6	5.1
2	1.7	9.2
2	1.8	14.4
2	1.9	25.4
2	2	43.4
