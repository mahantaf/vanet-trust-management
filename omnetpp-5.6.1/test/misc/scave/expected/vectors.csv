time,Net.subnet[0].offered load
0.1,30
0.2,40
0.3,10
0.4,20
0.5,50
0.6,60
0.7,70
0.8,80
0.9,90
1,100
1.1,110
1.2,120
1.3,130
1.4,140
1.5,150
1.6,160
1.7,170
1.8,180
1.9,190
2,200
time,Net.subnet[0].throughput (%)
0.1,35.11
0.2,45.06
0.3,8.12
0.41,22.96
0.51,53.21
0.6,59.88
0.7,65.34
0.8,69.81
0.9,73.47
1,76.46
1.1,78.91
1.2,80.92
1.3,82.57
1.4,83.91
1.5,84.5
1.6,83.92
1.7,80.86
1.8,74.87
1.9,68.77
2,57.16
time,Net.subnet[0].frames dropped (%)
0.1,0
0.21,0
0.31,0
0.41,0
0.51,0
0.6,0.2
0.7,0
0.8,0
0.9,1.2
1,0
1.1,0.9
1.2,1.4
1.3,2.1
1.4,2.2
1.5,2.6
1.6,5.1
1.7,9.2
1.8,14.4
1.9,25.4
2,43.4
